backend/baseclass.pm

```diff
@@ -991,6 +991,7 @@ sub new_ssh_connection {
         }
         else {
             bmwqemu::diag "Could not connect to $args{username}\@$args{hostname}, Retry";
+            bmwqemu::diag "Possible solutions: check network-settings e.g. VIRSH_CMDLINE or S390_NETWORK_PARAMS and VNC/SSH settings in there";
```

Comment:
Seems to me this will indeed print the string every time. I would move the inline condition to block one. I would also remove the "Possible solutions:" substring and just begin with "check network setttings..."